src/components/ListingData/ListingTable.js

```diff
@@ -0,0 +1,153 @@
+// Copyright (c)
```

Comment:
A part of the copyright header is missing here, it should be `// Copyright (c) Cosmo Tech.`